target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -131,6 +131,7 @@ kimax,u35wf)
 	;;
 kingston,mlw221|\
 kingston,mlwg2|\
+sitecom,wlr-4100v1002|\
```

Comment:
Wifi trigger can be moved to the DTS file.

Reply:
Sorry. I have no idea on how to embed it *.dts

Reply:
@polex wifi LED trigger in DTS like this...

```
		wlan2g {
			label = "blue:wlan2g";
			gpios = <&gpio3 0 GPIO_ACTIVE_LOW>;
			linux,default-trigger = "phy1tpt";
		};
```

change to "phy0tpt" for phy0
if it still does not work then do "phy0radio"

Reply:
In general, it's a good idea to look at other device files before/while compiling a PR ...